api/conf/settings/security.py

```diff
@@ -52,3 +54,11 @@
     USE_X_FORWARDED_HOST = True
     # https://docs.djangoproject.com/en/4.0/ref/settings/#secure-proxy-ssl-header
     SECURE_PROXY_SSL_HEADER = ("HTTP_X_FORWARDED_PROTO", "https")
+
+# Adding DJANGO_SECRET_KEY check
+if config("DJANGO_SECRET_KEY") == "example_key" and config(
+    "ENVIRONMENT", default="local"
+) not in ["local", "development"]:
+    raise ImproperlyConfigured(
+        "DJANGO_SECRET_KEY should not be 'example_key' in local or production environment."
+    )
```

Comment:
We should only check if `ENVIRONMENT != "local"`. We've historically used the word "development" to mean both local and staging environments, and it is critical that we continue to disambiguate these as distinct terms (by, in fact, leaving `development` behind entirely because it's too easy to confuse it's meaning). We have no other instances of checking "development" as an environment setting in any of our Python code, so we should not introduce it again here.

`ENVIRONMENT` is already defined in `BASE` and `SECRET_KEY` is already defined in this module (line 12), so we don't need to call `config` again to retrieve them. Doing so introduces multiple places that would need changing if those environment variable names ever changed.

The error message also needs a slight re-wording:

```suggestion
if SECRET_KEY == "example_key" and ENVIRONMENT != "local":
    raise ImproperlyConfigured(
        "DJANGO_SECRET_KEY must not be 'example_key' non-local environments."
    )
```

You'll need to add `ENVIRONMENT` to the list of imported variables from `conf.settings.base` on line 5.